### PR TITLE
Cypress/E2E: Update tests

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Update tests

#### Screenshots
Screen Shot 2022-01-20 at 8 13 39 AM
Screen Shot 2022-01-20 at 8 14 49 AM
Screen Shot 2022-01-20 at 8 16 25 AM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/keyboard_shortcuts/esc_close_modal_spec.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('Keyboard Shortcuts', () => {
     });
 
     it('MM-T1244 CTRL/CMD+K - Esc closes modal', () => {
-        const searchTerm = 'test';
+        const searchTerm = 'test' + Date.now();
```
Comment (saturninoabril) [inline e2e/cypress/integration/keyboard_shortcuts/esc_close_modal_spec.js:24]: Just to make a unique search and return no result

File: e2e/cypress/integration/team_settings/closed_team_invite_by_email_spec.js [hunk 2]
```diff
@@ -125,11 +125,6 @@ describe('Team Settings', () => {
         // * Check that the display name of the team the user was invited to is being correctly displayed
         cy.uiGetLHSHeader().findByText(testTeam.display_name);
 
-        // * Check that 'Town Square' is currently being selected
-        cy.get('.active').within(() => {
-            cy.get('#sidebarItem_town-square').should('exist');
-        });
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/closed_team_invite_by_email_spec.js:131]: This is no longer applicable once landed on "tips" page

File: e2e/cypress/support/index.js [hunk 3]
```diff
@@ -223,3 +218,13 @@ function sysadminSetup(user) {
...
 }
+
+function resetUserPreference(userId) {
+    cy.apiSaveTeammateNameDisplayPreference('username');
+    cy.apiSaveLinkPreviewsPreference('true');
+    cy.apiSaveCollapsePreviewsPreference('false');
+    cy.apiSaveClockDisplayModeTo24HourPreference(false);
+    cy.apiSaveTutorialStep(userId, '999');
+    cy.apiSaveOnboardingPreference(userId, 'hide', 'true');
+    cy.apiSaveCloudTrialBannerPreference(userId, '14_days_banner');
```
Comment (saturninoabril) [inline e2e/cypress/support/index.js:229]: Removed cloud trial banner to not show up whenever admin is used for testing

**Review by Phrynobatrachus — APPROVED** [review]
1 note otherwise LGTM

File: e2e/cypress/support/index.js (hunk 3 above)
Comment (saturninoabril) [reply e2e/cypress/support/index.js:229]: Oh I see. I think you're right. There's another option as "hide". I'll update this PR. Good catch! Thanks!

**Review by furqanmlk — APPROVED** [review]
Thanks @saturninoabril 👍

File: e2e/cypress/support/index.js (hunk 3 above)
Comment (saturninoabril) [reply e2e/cypress/support/index.js]: I've updated the command for consistency with API call. Unfortunately, "hide" is not being respected in some case like https://github.com/mattermost/mattermost-webapp/blob/master/components/announcement_bar/cloud_trial_announcement_bar/cloud_trial_announcement_bar.tsx#L140-L144. It's a separate issue to clarify though for Growth team.

**Pull request merged**